src/go/k8s/tests/e2e/regressions/01-call-controller-status.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: call-controller-status
+spec:
+  template:
+    spec:
+      containers:
+        - name: curl
+          image: badouralix/curl-jq:latest
+          env:
+            - name: POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+          command:
+            - /bin/sh
+            - -c
+          args:
+            - |
+              set -xe
+              # using the same value found in the duck-tape test https://github.com/redpanda-data/redpanda/pull/9737/commits/cf98f44141a5b07eddf5d2f7e69868585c7246f4
+              MAX_INDEX=1000
+
+              ADMIN_API_PATH="http://cluster-0.cluster.$POD_NAMESPACE.svc.cluster.local:9644"
+              CONTROLLER_STATUS_PATH="v1/debug/controller_status"
+              
+              # wait for admin server to be ready, should be serving quickly
+              CURL_CONTROLLER_STATUS_CMD="curl -so /dev/null -w "%{http_code}" ${ADMIN_API_PATH}/${CONTROLLER_STATUS_PATH}"
+              until [ "${status:-}" = "200" ]; do
+                status=$(${CURL_CONTROLLER_STATUS_CMD})
+                sleep 0.5
+              done
```

Comment:
This should be achieved by https://github.com/redpanda-data/redpanda/pull/9833/files#r1160057037

Reply:
Sure, i was just making sure we can talk to the api, but by the time the pod it is up it should be.